docker/shared/containers-list.sh

```diff
@@ -23,20 +23,11 @@ shopt -s nullglob globstar
 
 TOMCAT_IMAGE="tomcat"
 
-DEFAULT_TOMCAT_TAG="8.5-jdk8-openjdk-slim"
+DEFAULT_TOMCAT_TAG="9.0-jdk11-openjdk-slim"
 
 TOMCAT_DEBIAN_TAGS=(
     "8.5-jdk11-openjdk-slim"
-    "8.5.47-jdk11-openjdk-slim"
     "9.0-jdk11-openjdk-slim"
-    "9.0.27-jdk11-openjdk-slim"
-
-    "8.0-jre8-slim"
-    "8.5-jdk8-openjdk-slim"
-    "8.5.46-jdk8-openjdk-slim"
-    "9.0-jdk8-openjdk-slim"
+    "9.0-jdk13-openjdk-slim-buster"
```

Comment:
I am leaning towards Bullseye because:

We do install additional packages in our image: https://github.com/dhis2/dhis2-core/blob/ci/cleanup-docker-images-list/docker/tomcat-debian/Dockerfile#L19-L23

And the Tomcat image devs recommendation (extracted from the passage you quoted for clarity):

> If your image needs to install **any additional packages** beyond what comes with the image, you'll likely want to specify one of these **[sic: buster, bullseye]** explicitly to minimize breakage when there are new releases of Debian.

Granted, the packages we install seem safe, but even decade old assumptions in a stable distribution can be [upended between two releases](https://lwn.net/Articles/874247/), so, better safe than sorry.